source/administration/object-management/object-lifecycle-management.rst

```diff
@@ -10,41 +10,46 @@ Object Lifecycle Management
-tier. For object expiry, MinIO automatically deletes the object.
+MinIO Object Lifecycle Management allows creating rules for time or date based automatic transition or expiry of objects. 
+For object transition, MinIO automatically moves the object to a configured remote storage tier. 
+For object expiry, MinIO automatically deletes the object.
 
-MinIO lifecycle management is built for behavior and syntax compatibility with 
-:s3-docs:`AWS S3 Lifecycle Management <object-lifecycle-mgmt.html>`. For
-example, you can export S3 lifecycle management rules and import them into
-MinIO or vice-versa. MinIO uses JSON to describe lifecycle management rules,
-and conversion to or from XML may be required. 
+MinIO lifecycle management is built for behavior and syntax compatibility with :s3-docs:`AWS S3 Lifecycle Management <object-lifecycle-mgmt.html>`. 
+For example, you can export S3 lifecycle management rules and import them into MinIO or vice-versa. 
+MinIO uses JSON to describe lifecycle management rules, and conversion to or from XML may be required. 
 
 .. _minio-lifecycle-management-tiering:
 
 Object Transition ("Tiering")
 -----------------------------
 
-MinIO supports creating object transition lifecycle management rules, where
-MinIO can automatically move an object to a remote storage "tier". MinIO
-supports any S3-compatible service as a remote tier *in addition to* the
-following public cloud storage services:
+MinIO supports creating object transition lifecycle management rules, where MinIO can automatically move an object to a remote storage "tier". 
+MinIO supports any S3-compatible service as a remote tier *in addition to* the following public cloud storage services:
 
 - :ref:`Amazon S3 <minio-lifecycle-management-transition-to-s3>`
 - :ref:`Google Cloud Storage <minio-lifecycle-management-transition-to-gcs>`
 - :ref:`Microsoft Azure Blob Storage 
   <minio-lifecycle-management-transition-to-azure>`
 
-MinIO object transition supports use cases like moving aged data from MinIO
-clusters in private or public cloud infrastructure to low-cost private or public
-cloud storage solutions. MinIO manages retrieving tiered objects on-the-fly
-without any additional application-side logic. 
+MinIO object transition supports use cases like moving aged data from MinIO clusters in private or public cloud infrastructure to low-cost private or public cloud storage solutions. 
+MinIO manages retrieving tiered objects on-the-fly without any additional application-side logic. 
 
-Use the :mc:`mc admin tier` command to create a remote target for tiering
-data to a supported Cloud Service Provider object storage. You can then use the
-:mc-cmd:`mc ilm add --transition-days` command to transition objects to the
-remote tier after a specified number of calendar days.
+Use the :mc:`mc admin tier` command to create a remote target for tiering data to a supported Cloud Service Provider object storage. 
```

Comment:
Should this be 
> ... to a supported S3-compatible service.

?
As it, it seems like the command only works with the cloud providers.

Reply:
Supproted remote target would probably be fine since we have a list above.